config/php/docker-entrypoint.sh

```diff
@@ -62,148 +18,37 @@ function waitUntil() {
-    createAmqpVhost "${MESSENGER_TRANSPORT_CHANNEL_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_SEGMENT_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_COMPLETENESS_DSN}"
-    createAmqpVhost "${MESSENGER_TRANSPORT_NOTIFICATION_DSN}"
-
-    bin/console messenger:setup-transports --no-interaction import
-    bin/console messenger:setup-transports --no-interaction channel
-    bin/console messenger:setup-transports --no-interaction export
-    bin/console messenger:setup-transports --no-interaction core
-    bin/console messenger:setup-transports --no-interaction event
-    bin/console messenger:setup-transports --no-interaction segment
-    bin/console messenger:setup-transports --no-interaction completeness
-    bin/console messenger:setup-transports --no-interaction notification
+>&2 echo "Generate JWT keys"
+genereJwtKeys "config/jwt/private.pem" "config/jwt/public.pem"
 
-    >&2 echo "messenger initialization finished"
-fi
+>&2 echo "Composer install"
+#composer install --prefer-dist --no-progress --no-suggest --no-interaction
+#composer dump-autoload --optimize
+#composer dump-env dev
 
-if [ "$1" = 'php-fpm' ] ; then
-    createApplicationDirs
+>&2 echo "Waiting for db to be ready"
+waitUntil bin/console doctrine:query:sql "SELECT 1"
 
-    if [ "$APP_ENV" != 'prod' ]; then
-        composer install --prefer-dist --no-progress --no-suggest --no-interaction
-    fi
+>&2 echo "Build application"
+bin/phing build
 
-    >&2 echo "Waiting for db to be ready..."
-    waitUntil bin/console doctrine:query:sql "SELECT 1"
-
-    if [ "$APP_ENV" != 'prod' ]; then
-      bin/phing build
-    fi
-
-    bin/console ergonode:migrations:migrate --no-interaction --allow-no-migration
-
-    if [ "$APP_ENV" != 'prod' ]; then
-        enableXdebug
-        echo -e "\e[30;48;5;82mergonode  api is available at http://localhost:${EXPOSED_NGINX_PORT} \e[0m"
-    fi
-
-    >&2 echo "app initialization finished"
-fi
-
-if [ "$1" = 'crond' ] ; then
-    createApplicationDirs
-
-    function finish {
-      jobs=$(jobs -p)
-      if [[ -n ${jobs} ]] ; then
-        kill $(jobs -p)
-      fi
-    }
-    trap finish EXIT
-
-    # each enty for CRONTAB must be delimited by ; example:
-    # * * * * * /srv/app/bin/console channel:export:schedule >> /srv/app/var/log/crond.log ; * * * * * /srv/app/bin/console some:other:cmd >> /srv/app/var/log/crond.log
-
-    echo  "${CRONTAB:-* * * * * /srv/app/bin/console channel:export:schedule >> /srv/app/var/log/crond.log}"  | sed "s~;\s*~\n~g" > /tmp/cron.install
-    su -s /bin/bash www-data -c 'cat /tmp/cron.install | crontab -'
-    rm -f /tmp/cron.install
-    >&2 echo "installed crontab"
-    su -s /bin/bash www-data -c 'crontab -l'
-    su -s /bin/bash www-data -c "touch /srv/app/var/log/crond.log"
-    crond -f &
-    tail -f /srv/app/var/log/crond.log
-
-else
-  exec docker-php-entrypoint "$@"
-fi
+>&2 echo "Database migrations"
+bin/console ergonode:migrations:migrate --no-interaction --allow-no-migration
```

Comment:
Perhaps create DB as well if not exists

Reply:
Database creation it's done in `./config/postgres/init.sql`